utilities/data/getStoryblokRedirects.mjs

```diff
@@ -100,12 +100,13 @@ export const getStoryblokRedirects = async () => {
     return [];
   }
 
-  const redirects = stories.map(entry => ({
+  const redirects = stories
+  .filter(entry => entry.content.from && entry.content.to) // Filter out entries without "from" or "to"
+  .map(entry => ({
     source: sanitizeSourcePath(entry.content.from),
     has: extractQueryParameters(entry.content.from),
     destination: entry.content.to,
-    statusCode: sanitizeRedirectCode(entry.content.statusCode),
+    statusCode: sanitizeRedirectCode(entry.content.statusCode || 301),
```

Comment:
@sherakama Something interesting about creating redirects - Even though the `from` and `to` fields in Storyblok are required, and technically you can't save a redirect without entering something in those fields, the entry still exists without any values in those fields the moment it's created, so the empty test entry that you created in the demo today actually got "saved" and held up the Netlify build by causing an error 😆 Added this to filter out any entries without `to` and `from` fields, and add a fall back status code 301 here.

![Screenshot 2024-10-17 at 2 01 33 PM](https://github.com/user-attachments/assets/63d0c346-48d8-468d-bd35-7959d135896c)

Reply:
Oh that is kinda ridiculous. More Storyblok fun!

Reply:
I added an empty redirect to test and this PR preview built alright after I pushed up the commit to filter out those entries. I have now deleted that draft empty entry because it blocks my other PR preview 😂